scripts/build.d/pyside6

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+set -e
+
+SRCSYNC=${SRCSYNC:-git}
+
+if [ "${SRCSYNC}" == "tarball" ] ; then
+
+  display -e "\${SRCSYNC} ${SRCSYNC} not supported by this package"
+  exit
+
+elif [ "${SRCSYNC}" == "git" ] ; then
+
+  pkgname=pyside-setup
+  pkgbranch=${VERSION:-dev}
+  pkgfull=$pkgname
+
+  # unpack (clone)
+  syncgit https://code.qt.io/pyside ${pkgname} ${pkgbranch} ${pkgfull}
+
+else
+
+  display -e "Invalid \${SRCSYNC} ${SRCSYNC}"
+  exit
+
+fi
+
+# build.
+
+# prepare files for building.
+if [ ! -e ${DEVENVPREFIX}/bin/python3 ]; then
+  display -e "Python3 not exist!!"
+  exit
+fi
+
+PYTHON=${DEVENVPREFIX}/bin/python3
+
+if [ -z "${LLVM_INSTALL_DIR}" ] ; then
+  # set LLVM_INSTALL_DIR to select the libclang installation for shiboken.
+  # For example, a brew installed libclang may be specified with:
+  # export LLVM_INSTALL_DIR=/opt/homebrew/opt/llvm
+  echo "\${LLVM_INSTALL_DIR} is not set, shiboken might not find libclang"
```

Comment:
I guess we do not need it under a Ubuntu.  But for other systems it's useful to provide a way to find LLVM (libclang).